nextflow_schema.json

```
@@ -346,6 +346,19 @@
             },
             "fa_icon": "fas fa-tools"
         },
+        "bgc_gecco": {
+            "title": "BGC: gecco",
+            "type": "object",
+            "description": "",
```

```suggestion
            "description": "Biosynthetic gene cluster detection",
```
As in the description above for Antismash